requirements.txt

```diff
@@ -6,6 +6,12 @@ jsonschema==3.2.0
 lz4==3.0.2
 requests==2.23.0
 networkx==2.5
+six~=1.15.0
+python-dateutil==2.8.2
+frozendict~=2.1.0
```

Comment:
This dependency is not necessary, the same behavior can be done achieved with the standard library [MappingProxyType](https://docs.python.org/3/library/types.html#types.MappingProxyType)

Reply:
ok